test/hooks/useSelector.spec.tsx

```diff
@@ -81,17 +101,17 @@ describe('React', () => {
         })
 
         it('subscribes to the store synchronously', () => {
-          let rootSubscription
+          let rootSubscription: Subscription
 
           const Parent = () => {
             const { subscription } = useReduxContext()
             rootSubscription = subscription
-            const count = useSelector((s) => s.count)
+            const count = useSelector<DefaultRootState>((s) => s.count)
```

Comment:
:hand: Standard usage for `useSelector` is to either explicitly declare the type of `s`, or create a "pre-typed" version of the hook, per https://redux.js.org/usage/usage-with-typescript#define-typed-hooks .

Either of those is fine with me, but I'd appreciate changing over to those instead of using a generic in all of these.

Reply:
Yes, that's a great way to write it